examples/browser-to-server/tests/test.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable no-console */
+import { test, expect } from '@playwright/test'
+import { playwright } from 'test-util-ipfs-example'
+import { spawn, exec } from 'child_process'
+import { existsSync } from 'fs'
+
+// Setup
+const play = test.extend({
+  ...playwright.servers()
+})
+
+async function spawnGoLibp2p() {
+  if (!existsSync('../../examples/go-libp2p-server/go-libp2p-server')) {
+    await new Promise((resolve, reject) => {
+      exec('go build',
+        { cwd: '../../examples/go-libp2p-server' },
+        (error, stdout, stderr) => {
+          if (error) {
+            throw (`exec error: ${error}`)
+          }
+          resolve()
+        })
+    })
+  }
+
+  const server = spawn('./go-libp2p-server', [], { cwd: '../../examples/go-libp2p-server', killSignal: 'SIGINT' })
+  server.stderr.on('data', (data) => {
+    console.log(`stderr: ${data}`, typeof data)
+  })
+  const serverAddr = await (new Promise(resolve => {
+    server.stdout.on('data', (data) => {
+      console.log(`stdout: ${data}`, typeof data)
+      const addr = String(data).match(/p2p addr:  ([^\s]*)/)
+      if (addr !== null && addr.length > 0) {
+        resolve(addr[1])
+      }        
+    })
+  }))
+  return { server, serverAddr }
+}
+
+play.describe('bundle ipfs with parceljs:', () => {
+  // DOM
+  const connectBtn = '#connect'
```

Comment:
2 comments:

- is there no convention for constants, in many language it's something like `UPPER_CASE` but dunno about ts or PL in general. Either way I would expect it to be different from regular variable names?
- might want to use a convention suffix for these ones to make clear that these are Html Element Identifier strings rather than actual "buttons", "addresses", "input fields", etc...?

Reply:
>  convention suffix for these ones to make clear that these are Html Element Identifier strings rather than actual "buttons"

That's what the types are for :)

Reply:
What types? O.O

Reply:
You mean that this could be implied because it is of type str? Dunno, not entirely sure that is the case. Certainly not for the output one imho. I like to be very clear about this thing, but neither do I want to make a foothold here, so I'm fine with whatever you end up choosing here, just  note from my POV.

Reply:
Oh my mistake, I thought this was a `.ts` file

Reply:
This was a borrow from the WebTransport Playwright test, though Hungarian(ish) notation may be beneficial here.  I'm OK leaving as is.